runfile/trafficgen/trafficgen-remotehosts-runfile.json

```diff
@@ -0,0 +1,107 @@
+{
+  "tags": {
+    "topology": "RHEL10-ctx7-test-002pct"
+  },
+  "endpoints": [
+    {
+      "type": "remotehosts",
+      "settings": {
+        "user": "root"
+      },
+      "remotes": [
+        {
+          "engines": [
+            { "role": "client", "ids": [ 1 ] }
+          ],
+          "config": {
+            "host": "client-host.com",
+            "settings": {
+              "cpu-partitioning": true,
+	      "userenv": "alma8",
+	      "osruntime": "chroot"
+            }
+          }
+        },
+        {
+          "engines": [
+            { "role": "server", "ids": [ 1 ] }
+          ],
+          "config": {
+            "host": "server-host.com",
+            "settings": {
+              "cpu-partitioning": true,
+	      "userenv": "rhel95",
```

Comment:
Let's switch this to `alma8` since `rhel95` is a private userenv that people may not have access to.